resources/plugins/simln/charts/simln/templates/configmap.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "mychart.fullname" . }}-data
+data:
+  sim.json: |
+    {{ .Files.Get "files/sim.json" | nindent 4 }}
```

Comment:
instead of writing this file inside the chart, why not copy the process we use for scenarios: use an init container and then call `write_file_to_container()` to upload the data from an in-memory buffer?

Reply:
Updated to use `write_file_to_container`